### PR TITLE
brctl: add page;  id, history, kill: add examples;  kill: fix a typo 

### PR DESCRIPTION
**Pull request by mebeim**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform folder (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

---

- Add new page for the `brctl` linux command, a useful tool for creating and managing netwok ethernet interfaces.
- Add two examples for `id`:

    - The most basic example: `id`
    - An example showing how to check the id of a different user.

- Add one example for `history` on how to delete history entries.
- Add two examples for `kill` on how to work with multiple processes of the same group.
- Fixed a typo in the `kill` page:
  ```diff
  - pause a program, it until
  + pause a program until
  ```

### CODE REVIEW

**Comment by agnivade** [comment]
Hi @mebeim - Thanks for this. But in future, kindly create separate PRs for every page. So that when we review each page, the history belongs to that single page and we can easily choose to squash/rebase the commits.

Right now, it is not possible to squash the commits from the UI because it has unrelated commits for other pages.

File: pages/common/kill.md [hunk 1]
```diff
@@ -23,6 +23,14 @@
...
-- Signal the operating system to pause a program, it until a SIGCONT ("continue") signal is received:
+- Signal the operating system to pause a program until a SIGCONT ("continue") signal is received:
 
 `kill -{{17|STOP}} {{process_id}}`
+
+- Send a signal to all processes with the given GID (group id):
+
+`kill -{{signal_name}} -{{group_id}}`
+
+- Kill every process that can be killed by the current user (except init and kill itself):
```
Comment (agnivade) [inline pages/common/kill.md]: IMO, I don't think this example is necessary in a tldr page. Let us remove this.

File: pages/common/kill.md [hunk 2]
```diff
@@ -23,6 +23,14 @@
...
 `kill -{{9|KILL}} {{process_id}}`
 
-- Signal the operating system to pause a program, it until a SIGCONT ("continue") signal is received:
+- Signal the operating system to pause a program until a SIGCONT ("continue") signal is received:
 
 `kill -{{17|STOP}} {{process_id}}`
+
+- Send a signal to all processes with the given GID (group id):
+
+`kill -{{signal_name}} -{{group_id}}`
```
Comment (agnivade) [inline pages/common/kill.md]: Change signal_name to a concrete name like INT,KILL etc.

File: pages/common/id.md [hunk 3]
```diff
@@ -2,10 +2,18 @@
 
 > Display current user and group identity.
 
+- Display current user id (UID), group id (GID) and groups of which you are a member:
```
Comment (agnivade) [inline pages/common/id.md]: Display the current user's id (UID), group id (GID) and groups to which they belong.

File: pages/linux/brctl.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Add an interface to an existing bridge:
+
+`sudo brctl addif {{bridge_name}} {{interface_name}}`
+
+- Remove an interface from an existing bridge:
+
+`sudo brctl delif {{bridge_name}} {{interface_name}}`
+
+- Show a list with information about currently existing ethernet bridges:
```
Comment (agnivade) [inline pages/linux/brctl.md]: Let us make this the first example, since this is needed to get the bridge names.

**Comment by mebeim** [comment]
@agnivade thanks for pointing this out, I was not sure if multiple PRs would have been welcome or not. I'll apply the requested changes and keep your suggestion in mind for the future.

File: pages/common/id.md [hunk 5]
```diff
@@ -2,10 +2,18 @@
 
 > Display current user and group identity.
 
+- Display current user id (UID), group id (GID) and groups to which they belong:
```
Comment (agnivade) [inline pages/common/id.md]: user's

**Comment by agnivade** [comment]
Sure, no worries. But please do not force push. It makes it difficult for us to see what changed in a review.

**Comment by mebeim** [comment]
Makes sense, thanks!

File: pages/common/history.md [hunk 6]
```diff
@@ -13,3 +13,7 @@
 - Overwrite history file with history of current `bash` shell (often combined with `history -c` to purge history):
 
 `history -w`
+
+- Delete the history entry at the specified offset:
```
Comment (sbrl) [inline pages/common/history.md]: Is this an offset from the beginning or the end of the history?

File: pages/common/kill.md [hunk 7]
```diff
@@ -23,6 +23,10 @@
...
 `kill -{{9|KILL}} {{process_id}}`
 
-- Signal the operating system to pause a program, it until a SIGCONT ("continue") signal is received:
+- Signal the operating system to pause a program until a SIGCONT ("continue") signal is received:
 
 `kill -{{17|STOP}} {{process_id}}`
+
+- Send a `SIGUSR1` signal to all processes with the given GID (group id):
+
+`kill -{{SIGUSR1}} -{{group_id}}`
```
Comment (sbrl) [inline pages/common/kill.md]: Is the group id here numeric, or textual? If it's the latter, perhaps `group_name` would be better suited.

File: pages/common/history.md (hunk 6 above)
Comment (mebeim) [reply pages/common/history.md]: If you issue the `history` command, it becomes very clear:

```bash
marco:~/testdir$ history -c
marco:~/testdir$ ls
a  b  c
marco:~/testdir$ ls -l
total 0
-rw-r--r-- 1 marco marco 0 Jan 11 01:46 a
-rw-r--r-- 1 marco marco 0 Jan 11 01:46 b
-rw-r--r-- 1 marco marco 0 Jan 11 01:46 c
marco:~/testdir$ echo Hello
Hello
marco:~/testdir$ history
 1002  ls
 1003  ls -l
 1004  echo Hello
 1005  history
marco:~/testdir$ history -d 1003
marco:~/testdir$ history
 1002  ls
 1003  echo Hello
 1004  history
 1005  history -d 1003
 1006  history
marco:~/testdir$
```

File: pages/common/kill.md (hunk 7 above)
Comment (mebeim) [reply pages/common/kill.md]: It's the group ID, aka the `GID`, numerical.

File: pages/common/history.md (hunk 6 above)
Comment (sbrl) [reply pages/common/history.md]: Ah, I see :P I think I misinterpreted this. It's fine as-is then I think :-)

File: pages/common/kill.md (hunk 7 above)
Comment (sbrl) [reply pages/common/kill.md]: Ah, I see. Thanks for the clarification :-)

**Pull request merged**